load/issuer.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+
+	"github.com/ava-labs/hypersdk/api/ws"
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/pubsub"
+)
+
+var (
+	_ Issuer[*chain.Transaction] = (*DefaultIssuer)(nil)
+
+	ErrIssuedAlreadyStopped = errors.New("issuer already stopped")
+)
+
+type DefaultIssuer struct {
```

Comment:
(not necessary for this PR) This isn't so much a default as a HyperSDK specific issuer. We may want to put this in a different package or rename it within this one.